packages/valdueza-storybook/src/__snapshots__/Storyshots.test.js.snap

```diff
@@ -10755,7 +10744,6 @@ exports[`Storyshots Extension/Restore Wallet Set Password page 1`] = `
               placeholder="Confirm Password"
               required={true}
               type="password"
-              value=""
```

Comment:
I think you need to `yarn build` in medulas-react-components in order to avoid those unrelated diffs